app/assets/javascripts/pageflow/editor/models/page.js

```diff
@@ -48,7 +48,14 @@ pageflow.Page = Backbone.Model.extend({
   },
 
   title: function() {
-    return this.configuration.get('title') || this.configuration.get('additional_title');
+    var configurationTitle = this.configuration.get('title');
+    var title = configurationTitle || this.configuration.get('additional_title');
+
+    if(!title && configurationTitle === '') {
+      title = configurationTitle;
+    }
+
+    return title;
```

Comment:
So we want to ensure that `title` always returns a string and never `undefined`. Wouldn't it be easier to say:

```
return this.configuration.get('title') || this.configuration.get('additional_title') || '';
```